site/src/assets/data/faq/index.js

```diff
@@ -1,14 +1,70 @@
+                "Requiring users to sign up  and register a user account keeps the Playground a safe and healthy for all to enjoy. Without requiring users to sign-in the Meshery Playground would allow anonymous access for anyone and everyone to potentially perform nefarious acts. ",
+            ],
+        },
+        {
+            question: "Will I lose my work in the Cloud Native Playground?",
+            category: "Meshery Playground",
+            answer: [
+                "No, all of the designs that you have created in the Meshery Playground are saved to your user account and will be available to you each time that you sign into your account. Your designs will be available in the Meshery Playground and any your other Meshery deployments, too.",
+                "Yes, the Meshery Playground is reset on a daily basis. However, only deployments that you may have created will be lost (e.g.only MeshMap Visualizer is reset). All of your work in Meshery UI and MeshMap Designer will remain in your user account.",
+            ],
+        },
+        {
+            question: "Is Meshery open source?",
+            category: "Meshery Playground",
+            answer: [
+                "Yes, Meshery is a Cloud Native Computing Foundation (CNCF) project and is licensed under Apache v2.",
+                "As the extensible cloud native manager, Meshery is a pluggable platform, offering multiple extension points within which users and partners (ISVs) can customize and extend Meshery's functionality under the same or different license(s), offering commercial and non-commericial extensions.",
+            ],
+        },
         {
             question: "What is the difference between the Cloud Native Playground and MeshMap?",
             category: "Meshery Playground",
             answer: [
                 "The Cloud Native Playground (aka Meshery Playground) is a managed instance of Meshery that offers a sandbox environment in which half of MeshMap functionality is enabled (Designer mode) and the other half of MeshMap functionality is disabled (Visualizer mode).",
-                "The sandbox environment is not connected to an active Kuberentes cluster, and as such, specific actions within MeshMap Designer are also disabled. Meshery and MeshMap are feature-rich, sophisticated management applications for cloud native infrastructure. To access their full set of capabilities, simply deploy your own copy of Meshery into the environment of your choosing.",
+                "The sandbox environment is not connected to an active Kuberentes cluster, and as such, depending upon which Meshery Playgrouns environment you use specific actions within MeshMap Designer are also disabled. Meshery and MeshMap are feature-rich, sophisticated management applications for cloud native infrastructure. To access their full set of capabilities, simply deploy your own copy of Meshery into the environment of your choosing.",
```

Comment:
Before it was mentioned that `The Meshery Playground is connected to live Kubernetes cluster(s)` and right now it's said that it's sandbox. Could that be written in more clear way?

Reply:
Great catch. The first version of the Playground was disconnected from any live Kubernetes cluster. This new version _is_ connected. I'm updating this...